### DISCUSSION TITLE
Turn off screen backlight at the lowest brightness setting like how it is in Macbooks?

### DISCUSSION THREAD

**Discussion by ggjiza** [seerge/g-helper#892] [Ideas]
All I know is that it may require access to the BIOS.
I'm assuming that like the Fn-Lock function (enabled at BIOS level), the backlight level minimum could theoretically be set to practically off.

**Comment by seerge** [comment]
@ggjiza unfortunately that's not possible in a way you want it to my knowledge. 

You can "turn off" screen in windows, but that would lock screen as well (so you need to enter PIN again or at least press any key to unlock it). This is already possible via binding one of macro keys under Extra (Toggle Screen action)

**Reply by ggjiza** [reply]
I thought so, having been scrapping through the internet for any way to do it for the past year. Enabling the FN lock gave me some hope. Thanks for the input and this amazing tool.

**Reply by ghost** [reply]
Hello,
I also would have loved to see this option featured. 
I might be totally wrong but it seems possible after trying this program :
https://sourceforge.net/projects/turnoffmonitor/

Thanks a lot for Ghelper, it's an amazing program !
(edit: the link was bad)

**Reply by seerge** [reply]
@romainbtl from what i see that app simply draws a black rectangle over your screen:) It’s not the same as macbooks do - as they literally turn off screen backlight.

**Reply by ghost** [reply]
Damn I thought that was it. Sorry for wasting your time. 

Thanks again !
